networking.adoc

```diff
@@ -0,0 +1,1081 @@
+----
+
+## Configuring the SR-IOV Network Operator
+
+[source,terminal]
+----
+$ oc patch sriovoperatorconfig default \
+  --type=merge -n openshift-sriov-network-operator \
+  --patch '{ "spec": { "enableInjector": <value> } }'
+----
+
+
+[source,terminal]
+----
+$ oc patch sriovoperatorconfig default --type=merge \
+  -n openshift-sriov-network-operator \
+  --patch '{ "spec": { "enableOperatorWebhook": <value> } }'
+----
+
+
+[source,terminal]
+----
+$ oc patch sriovoperatorconfig default --type=json \
+  -n openshift-sriov-network-operator \
+  --patch '[{
+      "op": "replace",
+      "path": "/spec/configDaemonNodeSelector",
+      "value": {<node-label>}
+    }]'
+----
+
+## Using virtual functions (VFs) with DPDK and RDMA modes
+
+[source,terminal]
+----
+$ oc create -f intel-dpdk-node-policy.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f intel-dpdk-network.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f intel-dpdk-pod.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f mlx-dpdk-node-policy.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f mlx-dpdk-network.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f mlx-dpdk-pod.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f mlx-rdma-node-policy.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f mlx-rdma-network.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f mlx-rdma-pod.yaml
+----
+
+## Ingress Operator in {product-title}
+
+[source,terminal]
+----
+$ oc --namespace openshift-ingress create secret tls custom-certs-default --cert=tls.crt --key=tls.key
+----
+
+
+[source,terminal]
+----
+$ oc patch --type=merge --namespace openshift-ingress-operator ingresscontrollers/default \
+  --patch '{"spec":{"defaultCertificate":{"name":"custom-certs-default"}}}'
+----
+
+
+[source,terminal]
+----
+$ oc patch -n openshift-ingress-operator ingresscontroller/default --patch '{"spec":{"replicas": 3}}' --type=merge
+----
+
+
+[source,terminal]
+----
+$ oc create -f <name>-ingress-controller.yaml 
+----
+
+
+[source,terminal]
+----
+$ oc replace --force --wait --filename - <<EOF
+apiVersion: operator.openshift.io/v1
+kind: IngressController
+metadata:
+  namespace: openshift-ingress-operator
+  name: default
+spec:
+  endpointPublishingStrategy:
+    type: LoadBalancerService
+    loadBalancer:
+      scope: Internal
+EOF
+----
+
+
+[source,terminal]
+----
+$ oc -n openshift-ingress-operator patch ingresscontroller/default --patch '{"spec":{"routeAdmission":{"namespaceOwnership":"InterNamespaceAllowed"}}}' --type=merge
+----
+
+
+[source,terminal]
+----
+$ oc edit IngressController
+----
+
+
+[source,terminal]
+----
+$ oc edit IngressController
+----
+
+
+[source,terminal]
+----
+$ oc -n openshift-ingress-operator annotate ingresscontrollers/<ingresscontroller_name> ingress.operator.openshift.io/default-enable-http2=true
+----
+
+
+[source,terminal]
+----
+$ oc annotate ingresses.config/cluster ingress.operator.openshift.io/default-enable-http2=true
```

Comment:
Requires a declarative alternative

Reply:
@siamaksade do both of this and the preceding annotation require declarative statements? I added yaml for the one above this. Unless you think its necessary, I think its better to leave this at one TIP, instead of two.